tfidf-benchmark/NLP-pipeline-sklearn.ipynb

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                    with SimpleTimer() as timer:
</pre><p>We should put the overall workflow that follows in a function so that it's easier to follow. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/rapids-examples/pull/31/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>